spec/models/patient_spec.rb

```diff
@@ -1,8 +1,12 @@
 require "rails_helper"
 
 describe Patient, type: :model do
+  let(:reporting_timezone) { Period::ANALYTICS_TIME_ZONE }
   def refresh_views
+    original = ActiveRecord::Base.connection.execute("SELECT current_setting('TIMEZONE')").first["current_setting"]
+    ActiveRecord::Base.connection.execute("SET LOCAL TIME ZONE '#{Period::ANALYTICS_TIME_ZONE}'")
     LatestBloodPressuresPerPatientPerMonth.refresh
+    ActiveRecord::Base.connection.execute("SET LOCAL TIME ZONE #{original}")
```

Comment:
reset PG's time zone back to whatever it was originally, for safety

Reply:
The `LOCAL` in `SET LOCAL TIME ZONE` ensures that the change is only for the current transaction. Shall we rely on that behavior of Rspec to simplify this?

Reply:
Are you suggesting we make this a global RSpec hook?  I'd rather not make this change across our entire test suite, as I don't _think_ it would match production.